src/Eccube/Controller/Admin/Store/OwnerStoreController.php

```diff
@@ -259,13 +269,33 @@ public function apiInstall(Application $app, Request $request, $pluginCode, $ecc
         }
         $packageNames .= self::$vendorName.'/'.$pluginCode;
         $return = $this->composerService->execRequire($packageNames);
+
         if ($return) {
+            $url = $this->appConfig['package_repo_url'].'/collect';
+            // $composerMode = 1 is ComposerApi
+            $composerMode = 1;
+            if ($this->composerService instanceof ComposerProcessService) {
+                $composerMode = 2;
+            }
+            $data = array(
+                'code' => $pluginCode,
+                'version' => $version,
+                'download_date' => Carbon::now()->format('Y-m-d H:i:s'),
```

Comment:
Download date please update in package repos (if use it, please use `\DateTime`)